.github/workflows/rc.yml

```diff
@@ -37,9 +37,7 @@ jobs:
                           "https://api.github.com/repos/${{ github.repository }}/commits/$COMMIT_SHA")
           echo "author=$(echo $COMMIT_DETAILS | yq '.author.login')" >> $GITHUB_OUTPUT
   prepare:
-    needs: check_author
     runs-on: ubuntu-22.04
-    if: ${{ needs.check_author.outputs.latest_commit_author != 'svc-gh-is-01' }}
```

Comment:
we shouldn't remove this otherwise it would get into infinite loop everytime there is a push by svc user.